### PR TITLE
Fix/v0.5/ssh

### PR DESCRIPTION
**Pull request by rezamnkh79**

### CODE REVIEW

File: db/ssh.go [hunk 1]
```diff
@@ -171,6 +171,7 @@ func ConnectWithUserPass(myssh Ssh, commands []string) {
 	if err := session.Run(commandsInOneLine); err != nil {
 		log.Print("Failed to run: " + err.Error())
 	}
+	log.Print("\n", b.String())
 	_ = stdIn.Close()
 	_ = session.Close()
```
Comment (Amin-MAG) [inline db/ssh.go:176]: Delete this logs

File: db/ssh.go [hunk 2]
```diff
@@ -232,6 +233,7 @@ func ConnectWithSShKey(myssh Ssh, commands []string) {
 	if err := session.Run(commandsInOneLine); err != nil {
 		log.Println("Failed to run: " + err.Error())
 	}
+	log.Print("\n", b.String())
 	_ = stdIn.Close()
```
Comment (Amin-MAG) [inline db/ssh.go:237]: Delete this logs

File: docs/api/connection.md [hunk 3]
```diff
@@ -107,12 +117,14 @@ To get the id of a connection and check if there is a connection with this id fo
...
 - Request
     ```json
-    {
-        "data" : {"data":{"host":"37.152.181.64","password":"sko192j3h","username":"reza"},"id":2,"name":"ssh1"}
+     "data": {
+        "host": "37.152.181.64",
+        "password": "-----------",
+        "username": "reza"
     }
     ```
```
Comment (Amin-MAG) [inline docs/api/connection.md:129]: Open document of connection and compare it with other documents. Fix the issues. They should be JSON and wrapped by {}. Also cure about the tabs and spaces.

File: docs/api/connection.md [hunk 4]
```diff
@@ -30,13 +28,15 @@ To get the user connection with id.
...
-    }
+    "data": {
+        "host": "37.152.181.64",
+        "password": "---------------------",
+        "username": "reza"
+    },
+    "id": "34",
+    "name": "ssh9",
+    "type": "sshes"
+    }
```
Comment (Amin-MAG) [inline docs/api/connection.md:39]: Tab and spacing !!! compare with other documents.

**Review by Amin-MAG — CHANGES_REQUESTED** [review]
Thanks just fix small things, please.

File: docs/api/connection.md (hunk 3 above)
Comment (Amin-MAG) [reply docs/api/connection.md:129]: https://github.com/Ragnacodes/Legato-Backend/blob/fix/v0.5/ssh/docs/api/connection.md

File: db/ssh.go (hunk 2 above)
Comment (rezamnkh79) [reply db/ssh.go]: done

Comment (rezamnkh79) [reply db/ssh.go]: done

File: docs/api/connection.md (hunk 3 above)
Comment (rezamnkh79) [reply docs/api/connection.md:129]: done

**Review by Amin-MAG — APPROVED** [review]

**Pull request merged**